### ISSUE TITLE
Depecher is not working anymore from  4/12/21

### ISSUE THREAD

**Issue by carlosgonz0** [elros34/depecher#41]

**Comment by elros34** [comment]
Yes I am aware of it. Every telegram client which do not use tdlib >= 1.7.9 (probably in same cases 1.7.0 might be partially usable) was cut off few days ago.

Currently I thinking whether to try update whole code from tdlib 1.5 or switch to other client. Yottagram has interesting feature added and clear c without json conversion but have rough edes all over the place: crashes and ui bugs, similar to depecher several months ago. Also lack of up-to-date source code uploaded to work on it.

**Comment by carlosgonz0** [comment]
I know of Yottagram, but is too buggy, the Calls and Avatar not work to me, so i am using oficial Telegram at the moment than Yottagram. I hope that you upgrade Depecher to tdlib.1.7.9.

**Comment by carlosgonz0** [comment]
:  (
